### ISSUE TITLE
Applying a role to a single image

### ISSUE THREAD

**Issue by gkroon** [asciidoctor/asciidoctor-pdf#1376]
I wish to apply a role, that applies a small blue border in this case, to select images, but I can't figure out how.

My theme.yml:

```yml
role:
  blueborder:
    border-color: #8888ff
    border-width: 5
```

My failed attempts:

```adoc
.My caption.
[.blueborder]#image:picture.jpg[]#
```

```adoc
.My caption.
[.blueborder]
image::picture.jpg[]
```

```adoc
.My caption.
image::picture.jpg[role="blueborder"]
```

Am I doing something wrong, or is this not supported yet?

**Comment by mojavelinux** [comment]
That's no something that is currently supported. Custom roles only apply to phrases (spans of text).

**Comment by mojavelinux** [comment]
This is a duplicate of #1939 and will be resolved by the fix for that issue in the 2.0.0 release.
